frontend/src/components/common/ProfileSidebar.vue

```diff
@@ -22,6 +24,12 @@ const logout = () => {
     loading.value = true;
 };
 
+const showTermsBtn = () => {
+    toggleCloseble();
```

Comment:
It is better to know "what" is being toggled from `toggleClosable()`, the imported functions don't show the context is being toggled is the T&C dialog.
See the line 30, that one is better.